.github/actions/update-on-new-ipfs-tag/entrypoint.sh

```diff
@@ -17,6 +17,12 @@ echo "The latest IPFS tag is ${LATEST_IPFS_TAG}"
 if [ "$CURRENT_IPFS_TAG" = "$LATEST_IPFS_TAG" ]; then
     echo "http-api-docs already uses the latest go-ipfs tag."
 else
+     # update installation docs
+     while read -r file; do
+          echo "replacing $CURRENT_IPFS_TAG with $LATEST_IPFS_TAG in $file"
+          sed -E -i "s/$CURRENT_IPFS_TAG/$LATEST_IPFS_TAG/g" $file
```

Comment:
👍 Definitely limitations of not using a real [templating system](https://github.com/ipfs/ipfs-docs/pull/1081#discussion_r829998574).

My assumption is that these are unlikely to happen or might happen once, which will catch at review time.
Because we have the frontmatter requirement and we only transform `current version` to `new version`.
(if we document something about versionX specifically, it will be replaced only when we upgrade ipfs from versionX to versionY).

Reply:
Cool with me, thank you for the reply :)